elements/pfe-tabs/src/pfe-tabs.scss

```diff
@@ -8,76 +8,76 @@ $LOCAL: tabs;
 
 $pfe-tabs__panel--PaddingLeft-factor:  2;
 
-/// ===========================================================================
-/// DEFAULT TABS
-/// ===========================================================================
-
 :host {
   display: block;
+
+  .panels {
+    @include pfe-tabs-panel;
+  }
 }
 
 .tabs {
-  --pfe-tabs--BorderColor: #{pfe-color(surface--border)};
-
-  display: flex;
-  border: 0;
-  border-bottom: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor)};
+  @include pfe-tabs-tabset;
 }
 
-.panels {
-  padding: 0;
-  padding-top: #{pfe-var(container-padding)};
-}
 
 /// ===========================================================================
-/// VERTICAL TABS
+/// VERTICAL ORIENTATION
 /// ===========================================================================
 
 :host([vertical]) {
-  display: flex; // Makes the tabs side-by-side with panels
-}
+  display: flex; // Sets tabs to appear to the left of the panels.
 
-:host([vertical]) .tabs {
-  flex-direction: column;
-  width:  25%;
-  border: 0;
-  border-right: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor)};
-}
+  .tabs {
+    @include pfe-tabs-tabset-vertical;
+  }
 
-:host([vertical]) .tabs ::slotted(pfe-tab) {
-  margin: 0 -1px 0 0;
-  border: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} transparent;
-  border-right: 0;
-  position: relative;
+  .panels {
+    @include pfe-tabs-panel-vertical;
+  }
 }
 
-:host([vertical]) .tabs ::slotted(pfe-tab[aria-selected="true"]) {
-  border-color: #{pfe-local(BorderColor)};
-  border-right: 0;
+/// ===========================================================================
+/// TAB VARIANTS
+/// ===========================================================================
+
+:host([vertical][pfe-variant="earth"]) {
+  @include pfe-tabs-vertical;
+
+  .tabs {
+    @include pfe-tabs-tabset-vertical-earth;
+  }
 }
 
-:host([vertical]) .panels {
-  padding: 0;
-  padding-right: #{pfe-var(container-padding)};
-  padding-left: calc(#{pfe-var(container-padding)} * #{$pfe-tabs__panel--PaddingLeft-factor});
+:host([pfe-variant="earth"]) {
+  .tabs {
+    padding-left: #{pfe-var(container-padding)};
+
+    ::slotted(pfe-tab[aria-selected="false"]:not([vertical]):first-of-type) {
+      border-left: 0;
+    }
+  }
 }
 
-/// ===========================================================================
-/// TAB VARIANTS
-/// ===========================================================================
-:host([pfe-variant="primary"]) .tabs {
-  border-bottom: transparent;
-  border-right:  transparent;
+:host([on="dark"]) {
+  @include pfe-theme($theme: "dark");
+}
+:host([color="darker"]) {
```

Comment:
I don't think darker, darkest fit with the on attribute.  Can we remove these for now?  I don't think they're in use and they need more discussion?

Reply:
@castastrophe Commented out darker, darkest, lighter, lightest theme options and added TODOs.